app/Support/CatalogAuthorProjection.php

```
@@ -0,0 +1,117 @@
+                }
+                $stmt->bind_param(str_repeat('i', count($bind)), ...$bind);
+                $stmt->execute();
+                $stmt->close();
+            }
+        } catch (\Throwable $e) {
+            // Derived display data must never abort a book/author save. The
+            // legacy SELECT remains available until the projection is repaired.
+            SecureLogger::warning('CatalogAuthorProjection rebuild failed', [
+                'count' => count($ids),
+                'error' => $e->getMessage(),
+            ]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'App\Support\SecureLogger' in method 'rebuildMany'. (undefined)

(StaticAccess)